config.py

```diff
@@ -1479,9 +1505,76 @@ def get_sessions():
         return sessions_
 
 
-def get_runs() -> list:
+def get_runs_all_subjects() -> dict:
+    """Gives the mapping between subjects and their runs.
+
+    Returns
+    -------
+    a dict of runs present in the bids_path
+    for each subject asked in the configuration file
+    (and not for each subject present in the bids_path)."""
+    # We cannot use get_subjects() because if there is just one subject
```

Comment:
```suggestion
    (and not for each subject present in the bids_path).
    """
    # We cannot use get_subjects() because if there is just one subject
```